src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const stringModified = sourceString
+    .split(/;\s*/)
+    .filter((item) => item.trim() !== '');
+
+  stringModified.forEach((item) => {
+    const index = item.indexOf(':');
+
+    if (index !== -1) {
+      const objectKey = item.slice(0, index).trim();
+      const objectValue = item.slice(index + 1).trim();
+
+      if (resultObject[objectKey]) {
+        resultObject[objectKey] += ', ' + objectValue;
```

Comment:
The current implementation concatenates multiple values for the same property with a comma. This is not a standard way to handle multiple CSS declarations of the same property. Typically, the last one should overwrite the previous ones.